deployment/rpm/about.md

```diff
@@ -23,10 +23,10 @@ Biggest advantages of packaging your application is:
 Run:
 
 ```bash
-$ sudo dnf install fedora-packager rpmdevtools
+$ sudo dnf install fedora-packager rpmdevtools gcc
```

Comment:
I wonder why would you need gcc when you're building in mock if I'm not mistaken.

Reply:
You're not _only_ building in mock. § Quick Start Guide does end with a suggestion to run `rpmbuild -ba hello.spec`, and GNU Hello is a C program. 

Technically you'll need other build tools besides `gcc` to compile GNU Hello, tho: `make`, `autoconf`, etc. I don't _believe_ the three packages listed will necessarily pull all of those in, if they're not already installed.

Reply:
Ok, let's go with that, then.
Thanks!